index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
 
-As already mentioned, the structural requirements defined by the CSIP generally
-apply to the AIP. However, the AIP `representations` folder contains
-representations which are created as part of the AIP maintenance. If no
-maintenance operation was performed, this folder is not present. For this reason,
-the CSIP requirement which prescribes the presence of the `representations`
-folder is overruled by the following AIP specific requirement.
+If during the life-cycle of the AIP preservation actions are applied to specific parts, 
+i.e. child packages of the logical AIP, the AIC (header information package) must update
+the references to the child packages. However, it is not required to update the 
+reference to a parent of a child package which is not concerned by a preservation action.
 
-<a name="aip-representations-optional"></a>**AIP-REPRESENTATIONS-OPTIONAL**: The
-Archival Information Package (AIP) folder COULD include a folder named
-representations. This requirement overrules requirement *CSIPSTR9* defined by
-the CSIP. See also requirement *AIP-REPRESENTATIONS*.
+### Splitting 
+
+*Definition:* *Splitting* is a special case of segmentation where large files 
+(e.g. large representation content files) are divided into parts of a fixed byte 
+length. However, the splitted content files are wrapped by AIP segments, i.e. they 
+are contained in an AIP which references the parent information package (AIC) 
+to which they belong.
+
+## Differential AIP
+
+A differential package is an incomplete form of the AIP which contains only 
+part of the original AIP it is derived from. The purpose of the differential AIP
+is to allow persisting updates to a previously stored AIP.
+
+The differential AIP is mostly relevant for the physical container files
+storing the actual content of the AIP. In case of large AIPs, this allows
+adding or overriding data or metadata to an physical container
+containing parts of an AIP or the entire AIP content. 
+
+# AIP format
+
+The AIP format consists of a set of recommendations and requirements[^2]
+regarding the use of structural and preservation metadata which are 
+introduced in the following.
+
+## AIP specific structural metadata
 
 <a name="compdiv"></a>
 
 ### Compound vs. divided package structure
 
-As mentioned, the ability to manage representations or representation parts
-separately is required because the digital data submissions can become very
+The ability to manage representations or representation parts
+separately is required because the digital data submissions can be very
 large. This is not only relevant for storing the AIP, it also concerns the SIP
 which might need to be divided before the data is submitted to the repository.
-And regarding the DIP, it requires the capability to reconstruct the DIP from an
-AIP which is decomposed into multiple parts.
+And regarding the DIP, it concerns the capability to reconstruct the DIP from an
+AIP which is divided into multiple parts.
```

Comment:
Needs rewording as the meaning isn't clear.

Reply:
Reformulated: "In addition, it is important to find and identify AIP segments when creating a DIP which relies on metadata or content of these segments."